ipatests/test_webui/ui_driver.py

```diff
@@ -1150,6 +1150,33 @@ def get_row(self, pkey, parent=None, name=None):
                 return row
         return None
 
+    def get_row_by_column_value(self, key, column_name, parent=None, name=None):
```

Comment:
Please, change `name=None` to `table_name=None` to avoid confusion. Change is also needed on line 1158.

* Sorry for not fixing it right away in the example and taking your time now :-/

Reply:
change also needed on line 1173